### PR TITLE
feat: add Cadence Desktop

### PR DESCRIPTION
**Pull request by moringaman**
Cadence Desktop is a CDN Organizer and search tool is enables people who do not have access to the internet all the time to use css and javascript libraries in their projects via a local http server.

Website address is: https://www.cadence-desktop.com

#### Login
[Login-screen](https://firebasestorage.googleapis.com/v0/b/cadence-8edfc.appspot.com/o/login%20screen.gif?alt=media&token=d2bbaafc-82f6-486e-aed4-9765c672d5d5)

#### CDN Search
[Search-Screen](https://firebasestorage.googleapis.com/v0/b/cadence-8edfc.appspot.com/o/react_search.gif?alt=media&token=c0b4f637-09a7-421c-87b1-b8d0b5e74a29)

#### Markdown Note Creation
[Markdown-Image](https://firebasestorage.googleapis.com/v0/b/cadence-8edfc.appspot.com/o/readNotes.gif?alt=media&token=f07e4c04-9310-44ab-84b5-a6f476c8a61e)

Thankyou

### CODE REVIEW

File: apps/cadence-desktop/cadence-desktop.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: 'Cadence Desktop'
+description: 'Offline CDN http server, organiser and note taker.'
+website: 'https://www.cadence-desktop.com'
+category: ''Developer Tools'
```
Comment (Sanjit1) [inline apps/cadence-desktop/cadence-desktop.yml]: remove the ' before ```Developer Tools``` and make the line ```category: 'Developer Tools'```
That should fix the CI error

Comment (moringaman) [reply apps/cadence-desktop/cadence-desktop.yml]: Thanks Sanjit, this has now been done.

**Comment by Sanjit1** [comment]
So you seem to be set!
But for some reason no ones build has passed

Everyone's build is `expected`
So you may need  to wait

https://img.shields.io/badge/Keep%20Koding%2C%20Best%20regards-Sanjit-accff2

**Comment by moringaman** [comment]
Please see blow image of dev environment.
cadence_is_electron
I hope this proves that it is made with electron

**Comment by moringaman** [comment]
Please can I get this reviewed again, somebody?

**Comment by moringaman** [comment]
@HashimotoYT, can you please review this for me, thanks

**Comment by moringaman** [comment]
I'm not sure how long I will have to wait for this to be reviewed but I think I've been really patient @HashimotoYT

**Comment by Sanjit1** [comment]
> I'm not sure how long I will have to wait for this to be reviewed but I think I've been really patient @HashimotoYT

I feel you, man.
ping @HashimotoYT 

[best-regards](https://www.sanjit.wtf)

**Comment by moringaman** [comment]
> As per https://github.com/electron/apps/blob/master/contributing.md#company-logos-and-names --
> 
> If Cadence has given you permission to use that name, please provide that information in the PR so that the app's name can be used as-is. Otherwise,
> 
> > Please do not directly use another company's name or product without permission. It's generally better to refer to it in a dependent clause; for example, after "compatible with", "on", or "for."
> > For example, while we would not accept a third-party app named "GitHub Notifications", we would consider "Yourname Notifications for GitHub".

@HashimotoYT - The requested change has now been made please review

**Pull request merged**